packaging/datadog-agent/source/setup_agent.sh

```diff
@@ -389,18 +403,21 @@ tar -xz -C "$DD_HOME/agent" --strip-components 1 -f "$DD_HOME/agent.tar.gz"
 rm -f "$DD_HOME/agent.tar.gz"
 print_done
 
+# get the version from the actual config file in the branch
+AGENT_VERSION=$(cat $DD_HOME/agent/config.py | grep AGENT_VERSION | cut -f2 -d'=')
```

Comment:
hmm I understand the idea but to me `AGENT_VERSION` should always be the version that the user requested no?

The only places where we need a "resolved" agent version (i.e. the one in `config.py`) instead of the one the user requested is when we compare the version with known versions to determine if/how to install jmxfetch/integrations.

Further, if I'm using `AGENT_VERSION="oli/foo"`, I'd expect `INTEGRATIONS_VERSION` to default to `"oli/foo"` too, not to the tag from `config.py`.

I don't really mind either way, this is really a nitpick, feel free to ignore